lib/screens/home/chat/message/message_tile.dart

```diff
@@ -266,7 +268,9 @@ class _MessageTileState extends State<MessageTile> {
           color: const Color(0x00000000),
           icon: Icons.content_copy,
           foregroundColor: Colors.grey,
-          onTap: () => Clipboard.setData(ClipboardData(text: widget.message.text)),
+          onTap: () {
+            copyToClipboard(widget.message.text).show(context);
```

Comment:
I think it is better if you do a function like that `copyToClipboard(context, widget.message.text);` (with `context` as the first argument)